### PR TITLE
Cymerrad/etheratom

### PR DESCRIPTION
**Pull request by cymerrad**
This is the change I wanted to make a while back.  
It simplifies usage of the packages, by exporting everything from imported package and its dependencies.  
Look at the examples for what I mean by that.  
  
Changes are bundled with etheratom PR, because there are some exporting nuances involved, that I had to solve. This will also make my etheratom and remix-ide PR's more pleasant to the eye.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,9 +41,7 @@ resolver
 ```typescript
 import { parsers, resolvers, ResolverEngine } from "@resolver-engine/core";
 
-const resolver = new ResolverEngine<string>()
-  .addResolver(resolvers.UriResolver())
-  .addParser(parsers.UrlParser());
+const resolver = new ResolverEngine<string>().addResolver(resolvers.UriResolver()).addParser(parsers.UrlParser());
```
Comment (ritave) [inline README.md]: seems less readable?

File: packages/imports-fs/src/index.ts [hunk 2]
```diff
@@ -1,3 +1,11 @@
+export { Context, firstResult, Options, ResolverEngine, SubParser, SubResolver } from "@resolver-engine/core";
+export {
```
Comment (cymerrad) [inline packages/imports-fs/src/index.ts]: Those explicit exports are what I called a 'nuance'. Without it, some build tool called Rollup, can't find the `gatherSources` function.

File: README.md (hunk 1 above)
Comment (squadack) [reply README.md]: I second that. Imho it looked better before.

Comment (cymerrad) [reply README.md]: I concurr, however this is what Prettier insists on having

**Pull request merged**